federation/pkg/kubefed/util/util.go

```diff
@@ -115,7 +115,7 @@ func CreateKubeconfigSecret(clientset *client.Clientset, kubeconfig *clientcmdap
 		return nil, err
 	}
 
-	// Build the secret object with the minified and flattened
```

Comment:
Revert back this fix. This is indeed "minified". Yes, spelling correction flags this because it is not a word in the dictionary, but is in the tech jargon. See `--minify` flag here - http://kubernetes.io/docs/user-guide/kubectl/kubectl_config_view/